### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by WadeForward**
* Added Commands write up
* Added `Natural Date & Time` section under command format
* Updated FAQ with a link to `Date & Time format`
* Added glossary
* Updated command diagrams in user guide
* Updated PPP

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -524,19 +559,35 @@ Action | Format, Examples
...
+--------------------------------------------------------------------------------------------------------------------
+
+
+## Glossary
+
+This glossary provides definitions for the special terms used in the user guide.
+
+**Index:** The number on the left side of the corresponding entry in the patient or appointment book.
+
+**Tag:** A word or phrase that labels a patient.
```
Comment (YangYue128-helen) [inline docs/UserGuide.md]: I think it would be more clear to say that a tag indicates the medical condition of a patient. What do you think?

**Review by YangYue128-helen — APPROVED** [review]
Other than a minor phrasing issue, LGTM! 
We can see whether your images look ok after merging.

File: docs/UserGuide.md (hunk 1 above)
Comment (WadeForward) [reply docs/UserGuide.md]: Good point!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -524,19 +559,35 @@ Action | Format, Examples
...
+
+## Glossary
+
+This glossary provides definitions for the special terms used in the user guide.
+
+**Index:** The number on the left side of the corresponding entry in the patient or appointment book.
+
+**Tag:** A word or phrase that indicates the medical condition of a patient.
+
+**GUI:** A acronym that stands for Graphic User Interface.
```
Comment (JinHao-L) [inline docs/UserGuide.md]:
```suggestion
**GUI:** An acronym that stands for Graphic User Interface.
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -79,13 +79,13 @@ This section introduces ...
 
 ## Commands
 
-This section introduces the various commands used in Nuudle.
+The command is the main way to schedule appointment for a patient with Nuudle.
 
-[general commands](#general-commands), [patient management](#patient-management), [command summary](#command-summary)
+This section provides information about the commands on [general commands](#general-commands), [patient management](#patient-management), [command summary](#command-summary) accordingly.
```
Comment (JinHao-L) [inline docs/UserGuide.md]: Missed out on the `appointment-management` here.

Comment (JinHao-L) [inline docs/UserGuide.md]: This intro feels abit off. Some suggestion that might help:

This section introduces the various commands used in Nuudle

There are three main command types in Nuudle: 
1. General Commands
2. Patient Management Commands
3. Appointment Management Commands

An overview of all Nuudle's commands is also provided in the section below

**Review by JinHao-L — APPROVED** [review]
Other than some minor details, the rest LGTM 👍

File: docs/UserGuide.md (hunk 2 above)
Comment (WadeForward) [reply docs/UserGuide.md]: thanks for the spot

File: docs/UserGuide.md (hunk 3 above)
Comment (WadeForward) [reply docs/UserGuide.md]: thanks for the spot!

Comment (WadeForward) [reply docs/UserGuide.md]: thanks for the suggestion and I did changed it

**Pull request merged**